src/containers/SignUp/SignUp.js

```diff
@@ -12,36 +12,32 @@ class SignUp extends Component {
 
     state = {
         didSignUp: false,
-        state: null,
         district: null,
         communicationMethods: null,
         postCallerDetailsError: null
     }
 
     handleFormSubmit = (fieldsValues) => {
-        const {congressionalDistrict} = {...fieldsValues};
-        const state = congressionalDistrict[0];
-        const district = congressionalDistrict[1];
-        
+        const {district, zipCode, firstName, lastName, phone, email, } = {...fieldsValues};
```

Comment:
You don't need to spread an object right before doing destructing assignment:
 
```suggestion
        const { district, zipCode, firstName, lastName, phone, email } = fieldsValues
```